etl/steps/data/garden/ggdc/2022-11-28/penn_world_table.meta.yml

```diff
@@ -0,0 +1,175 @@
+dataset:
+  namespace: ggdc
+  version: 2022-11-28 
+  title: Penn World Table 10.0
+  short_name: penn_world_table
+  publication_year: 2021
+  description: Penn World Table version 10.0 is a database with information on GDP and its composition, employment, productivity and trade, covering 183 countries between 1950 and 2019.
+  sources:
+    -
+      name: Penn World Table 10.0
+      published_by: Feenstra, R. C., Inklaar, R. and Timmer, M.P. (2015), "The Next Generation of the Penn World Table". American Economic Review, 105(10), 3150-3182
+      publication_year: 2021
+      date_accessed: 2022-11-28
+      url: https://www.rug.nl/ggdc/productivity/pwt/
+
+tables:
+  penn_world_table:
+    variables:
+      rgdpe:
+        title: Expenditure-side real GDP at chained PPPs
+        unit: Million international-$ at 2017 prices
```

Comment:
I think we usually present unit and short unit in lower case (e.g. "million...").

Reply:
Thanks for that, anyway I will change this to just int-$, adding my transformations on this step as discussed in the other comment.